### PR TITLE
Auto-set topics of the -stats channels to indicate what they are and where to go to complain

### PR DESCRIPTION
**Pull request by royrapoport**

### CODE REVIEW

**Comment by jdq22** [comment]
Is this setting the topic every week for every -stats channel?

**Comment by royrapoport** [comment]
Yes!

**Comment by jdq22** [comment]
If you feel up to it, could you restrict setting the topic if there's no change to be made?

**Comment by royrapoport** [comment]
I can totally do that.  What's your thinking around why?

(It wouldn't limit API calls, just either keep them at the same number or increase them)

**Comment by jdq22** [comment]
I find it annoying to see the topic get changed, only for there to be no changes.

**Comment by royrapoport** [comment]
Oh! yes, that's a good reason to try to avoid having that happen.  I didn't think of that!
Hold on, I'll make that change.

**Comment by royrapoport** [comment]
@jdq22 Good suggestion.  Integrated into the PR.

File: slacker.py [hunk 1]
```diff
@@ -18,6 +18,26 @@ def __init__(self, slack, token):
...
+        """
+        Set the topic for cid it it's not already the expected topic
+        """
+        current_topic = self.get_topic(cid)
+        if current_topic != topic:
+            self.set_topic(cid, topic)
+
+    def get_topic(self, cid):
+        response = self.api_call("conversations.info?channel={}".format(cid))
+        return response .get("channel", {}).get("topic", {}).get("value")
```
Comment (jdq22) [inline slacker.py]: Is that a space between response and .get?

Comment (royrapoport) [reply slacker.py]: Yes.  Fixed.

**Review by jdq22 — APPROVED** [review]

**Pull request merged**